en/docs/deploy/aws.md

```diff
@@ -1,36 +1,177 @@
-# Deploying WSO2 Identity Server on Kubernetes using AWS
+---
+template: templates/single-column.html
+---
 
-## Step 1 - Checkout this repository into your local machine using the following Git command
+# Deploying WSO2 Identity Server using AWS CloudFormation
 
-```java
-git clone https://github.com/wso2/aws-is.git
+## Prerequisites
+
+1. Make sure you already have an [AWS account](https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/). 
+
+2. Install [AWS CLI 2](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html). Alternatively, you can also use AWS CLI version 1. However, you need to make sure that you have installed Python3 to use AWS CLI version 1.
+
+## Step 1 - Create and upload an SSL certificate into AWS
+In AWS, web servers are fronted with a Load balancer. While deploying the WSO2 Identity Server in AWS, we need not create the load balancer separately since it is taken care of in the template we use. For a secure connection via the load balancer between the web server and the browser, we need an SSL certificate. 
+
+??? note "Creating a self signed certificate"
+	If you are using this for testing purposes and do not want to create a certificate using AWS certificate manager, you can create a self signed certificate instead by following the instructions given below. 
+
+	1. Generate private key as private.pem 
+
+		**Request**
+
+		```curl 
+		openssl genrsa -out private.pem 2048
+		```
+		
+		**Response**
+
+		```curl
+		Generating RSA private key, 2048 bit long modulus
+		...................................................................................+++
+		......................................................................................+++
+		e is 65537 (0x010001)
```

Comment:
The requests and responses can go into tabbed code blocks. It might look better on the page. WDYT?